src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -230,6 +177,10 @@ You can view real-time events related to the sticker context.
 * *Dynamic Receipt* – Debug the receipts that are sent to the buyer and seller, when in-app purchases are implemented with dynamic receipts.
 You can view real-time events related to the receipts.
 
+TIP: To preview a custom context, you can override the environment settings to append the parameter to the URL.
```

Comment:
I'm not sure I understand this - is this about showing different contexts inside the debugger? Is this not solved by changing the tabs in the top right?

Reply:
Thanks! The previous version of the page talked about how to test a custom context (vs. a built-in e.g. sticker) in the live player. Is that still an available test mode, or should we just remove this tip if it's confusing?